components/Common/CodeTabs/index.tsx

```diff
@@ -23,6 +23,7 @@ const CodeTabs: FC<PropsWithChildren<CodeTabsProps>> = ({
   <Tabs
     {...props}
     className={styles.root}
+    triggerClassName={styles.trigger}
```

Comment:
I don't think we need to add a new prop called "triggerClassName" you can achieve the same by overriding the Tab styles with !important (`!`) I know in general that's not the best thing, but in this case it makes sense.

Reply:
The `!important` directive also works. But I've chosen to use the className so that the css is explicit (it says the class trigger to this style).

Reply:
It's more like, this feels very specific. But I'm fine either ways.